src/main/java/org/ibp/api/java/impl/middleware/ontology/validator/VariableValidator.java

```diff
@@ -389,6 +397,18 @@ private boolean scaleDataTypeValidationProcessor(final VariableDetails variable,
 				}
 			}
 
+			final boolean isFileType = Objects.equals(scale.getDataType(), DataType.FILE_VARIABLE);
+			// Temporally restriction for Germplasm Attributes and Passport Variable Types.
```

Comment:
Fix typo: Temporally to temporal
Add reference to IBP-4798, this restriction should be removed when implementing this ticket.

Reply:
done